xiidea/easy-audit/1.4/post-install.txt

```diff
@@ -0,0 +1,3 @@
+<bg=blue;fg=white> Next for XiideaEasyAuditBundle </>
+
+* <fg=blue>Read</> the documentation at <comment>http://xiidea.github.io/EasyAuditBundle/</>
```

Comment:
I would remove this file altogether as the user probably comes from the docs anyway. We try to limit those post install messages as much as possible.

Reply:
@fabpot File removed as suggested.